uint8 layer
uint32 key
uint64 cfgData
---
bool got_Ack
bool got_Nack